src/components/BookDetail/BookDetail.jsx

```diff
@@ -63,6 +66,10 @@ function BookDetail() {
                         <p>{bookPageCount}</p>
                         <p>{bookGenre}</p>
                         <p>{bookBlurb}</p>
+
+                        <Reviews reviews={reviews} />
+
+
```

Comment:
maybe remove some of these whitespaces as well

Reply:
Done, whitespaces removed.